article_store/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6.4-jessie
+
+RUN pip install pipenv
+
+ADD . /srv/app/
+WORKDIR /srv/app/
+
+RUN pipenv install --system
```

Comment:
does this mean system-wide as in overwriting the Python packages provided by the container's distribution? I'd be more cautious to avoid interaction between non-project file and project files, if it's just a default choice?

Reply:
I was just following on from a few example images such as [this](https://hub.docker.com/r/kennethreitz/pipenv/~/dockerfile/). I don't really have a preference or have had enough `docker` + `pipenv` experience yet. Happy to work with whatever is decided.

Reply:
Tried without `--system` in another PR and it seems to work correctly, I'd go for that as the conservative option

Reply:
OK Done 👍